script/staging/StagingDeploymentBase.sol

```diff
@@ -178,7 +180,18 @@ contract StagingDeploymentBase is Script {
         usds    = mainnet.config.readAddress(".usds");
         susds   = mainnet.config.readAddress(".susds");
         daiUsds = mainnet.config.readAddress(".daiUsds");
-        psm     = mainnet.config.readAddress(".psm");
+        livePsm = mainnet.config.readAddress(".psm");
+
+        // This contract is necessary to get past the `kiss` requirement from the pause proxy.
+        // It wraps the `noFee` calls with regular PSM swap calls.
+        psm = address(new PSMWrapper(USDC, dai, livePsm));
+
+        // NOTE: This is a HACK to make sure that `fill` doesn't get called until the call reverts.
+        //       Because this PSM contract is a wrapper over the real PSM, the controller queries
+        //       the DAI balance of the PSM to check if it should fill or not. Filling with DAI
+        //       fills the live PSM NOT the wrapper, so the while loop will continue until the
+        //       function reverts. Dealing DAI into the wrapper will prevent fill from being called.
+        IERC20(dai).transfer(psm, USDS_UNIT_SIZE * 10);
```

Comment:
This seems like a large amount to burn each time. If unit size is $10 then this will burn $100 every time. Does it need to be this big?

Reply:
You're right, setting it equal to unit size is sufficient to pass the check. Updated.